ONSdigital/dp-frontend-renderer
# Feature/census survey banners

### What

This PR adds the markup for the new promotional tiles - specifically the census and survey banners.

Currently the promotional tiles are in the left column on the homepage. First iteration will have them on the right, but for now I only have a placeholder markup for the left side column. I believe once the Population tile is added in, the columns should align properly and the promotional tiles should move to the right side

![image](https://user-images.githubusercontent.com/23668262/80725113-8f5e4b00-8afa-11ea-9995-6f0b739ad147.png)

### How to review

- Check that the styles align with requirements
- You can view the banners themselves locally by pulling `feature/census-survey-banners` in `dp-frontend-renderer`, ensuring that `NEW_HOMEPAGE_ENABLED=true` in `dp-frontend-router` and running `dp-homepage-controller`.
- Check that the copy aligns with requirements.

### Who can review

Anyone but me
